src/map/script.c

```diff
@@ -25305,6 +25305,75 @@ static BUILDIN(openrefineryui)
 	return true;
 }
 
+/**
+ * identify(<item id>)
+ * Identifies the first unidentified <item id> item on player's inventory.
+ * Returns -2 on error, -1 if no item to identify was found, identified idx otherwise.
+ */
+static BUILDIN(identify)
+{
+	struct map_session_data *sd = script_rid2sd(st);
+
+	if (sd == NULL) {
+		script_pushint(st, -2);
+		return true;
+	}
+
+	int itemid = script_getnum(st, 2);
+	if (!itemdb->exists(itemid)) {
+		ShowError("script_identify: Invalid item ID (%d)\n", itemid);
```

Comment:
please use `builtin_` for function name, just to be consistence with the reset of the commands.